src/views/MatchDetail.vue

```diff
@@ -249,6 +235,13 @@ export default class MatchDetailView extends Vue {
     );
   }
 
+  get matchIs4v4() {
```

Comment:
Doesn't seem like this is used anymore. You can delete it

Reply:
f72cd75 done 👍

Reply:
09eccd4 done 👍